example/spec/parallel/testAPI/scheduledRuleWithExecutionPrefixSpec.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const {
+  addCollections,
+  cleanupCollections,
+  waitForTestExecutionStart,
+} = require('@cumulus/integration-tests');
+
+const rulesApi = require('@cumulus/api-client/rules');
+const { deleteExecution } = require('@cumulus/api-client/executions');
+
+const { randomId } = require('@cumulus/common/test-utils');
+
+const {
+  createTestSuffix,
+  createTimestampedTestId,
+  loadConfig,
+  timestampedName,
+} = require('../../helpers/testUtils');
+
+describe('When I create a scheduled rule with an executionNamePrefix via the Cumulus API', () => {
+  let config;
+  let execution;
+  let executionName;
+  let executionNamePrefix;
+  let scheduledRuleName;
+  let scheduledHelloWorldRule;
+  let testSuffix;
+  let executionArn;
+  let beforeAllFailed;
+
+  const collectionsDir = './data/collections/s3_MOD09GQ_006';
+
+  beforeAll(async () => {
+    try {
```

Comment:
For posterity, try/catch with beforeAll failure check boilerplate was *added* as well (this is A Good Thing(tm)  👍🏻